src/DPOT_SOLIDIFY/dpot_test.cpp

```diff
@@ -19,35 +16,12 @@ lib_deps =
 */
 
 // Setting integers to for data keepers
-int num_average = 10; //The number of counts before the average is taken
 int sensing_pin_op_amp = A0; //Where the analog integer will be housed
 int slave_select_digi = 9; // Slave selected pin for the arduino (can change depending on board)
-int serial_speed = 9600;
-
-//Now setting datastore structure and SD Mechanics
-
-struct datastore //Now setting datastore structure 
-
-{
-// The datatype followed by the name of the place holder is given
-
-unsigned long int millistime; //Time from the millis function count
-unsigned long int microtime; // Time from the smaller micros function
-float Voltage_analog_input; // The analog input single from the OD sensor
-int digi_pot_wiper_position; //The digipot wiping position
-};
-
-File dataFile; // Created an instance of the open file 
-
-const int datalen = 20; //Size off each struct or... amount of time it takes to dump data to SD NOTE MUST BE CONST INT
-char *filename = "OD_Test_Solid.dat"; // The name of the file
```

Comment:
again, this is deleted because it's not used.  also, it's not formatted

Reply:
It was used?  That is how I printed the structure to the serial display